lenisha/scribe
# [TELUSHAS-62][TELUSHAS-45] - Update text, Update branding

### 📝 Description

- Updated text that Rebecca said should be changed
- Removed unused images
- Added TELUS logo files
- Added TELUS favicon
- Added TELUS logo to sidebar 

### 🔗 Link to JIRA Ticket

[Update text](https://willowtree.atlassian.net/browse/TELUSHAS-62)
[Branding update](https://willowtree.atlassian.net/browse/TELUSHAS-45)

### 🧪 Testing Notes

- Make sure the text is updated
- Make sure the favicon is updated
- Make sure the TELUS Health logo is on the sidebar

### 📸 Screenshots / 🎥 Video

- Sidebar logo

![Screenshot 2024-12-19 at 3 14 13 PM](https://github.com/user-attachments/assets/9d404fc9-d204-4b9f-a2d4-29758d82b8b9)

- Favicon

![Screenshot 2024-12-19 at 3 14 50 PM](https://github.com/user-attachments/assets/6067ae0c-4527-4340-8716-f8e1aa689826)

- Text

![Screenshot 2024-12-19 at 3 15 22 PM](https://github.com/user-attachments/assets/3072f9cc-a120-441b-8473-a4720456334f)
